packages/maskbook/src/plugins/NFT/Worker/index.ts

```diff
@@ -0,0 +1,8 @@
+import type { Plugin } from '@dimensiondev/mask-plugin-infra'
+import { base } from '../base'
+
```

Comment:
It seems this Worker is not doing anything. Did you forget to import its RPC to start it in the Worker?

Reply:
Removed.

Reply:
did you commit?